helx/charts/nginx/templates/ingress.yaml

```diff
@@ -25,4 +25,32 @@ spec:
             name: {{ include "nginx.fullname" . }}
             port:
               number: {{ .Values.service.httpPort }}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "nginx.fullname" . }}-admin
+  annotations:
+    nginx.ingress.kubernetes.io/whitelist-source-range: "{{ .Values.ingress.admin_ip_whitelist }}"
+  {{- with .Values.ingress.annotations }}
+    {{- toYaml . | nindent 4 }}
```

Comment:
Think you need that `annotations:` line, copied from above:
```
{{- with .Values.ingress.annotations }}
  annotations:
    {{- toYaml . | nindent 4 }}
{{- end }}
```

Reply:
The `annotations` is up two lines in this block. The `Values.ingress.annotations` is applied to both ingress definitions, and I don't think it would be desirable to add the whitelist annotations to both. We could split into three blocks, `Values.ingress.common.annotations`, `.Values.ingress.root.annotations` and `.Values.ingress.admin.annotations` to explicitly make all three contextual values

Reply:
Ahh good point.

I have a slight preference for leaving `.Values.ingress.annotations` as only effecting the main ingress, and instead just adding the new `.Values.ingress.admin.annotations`. That way we don't have to update any values.yaml for any helx instances that have already migrated to Ingresses.

Would be trading a few lines of duplicated code for 100% backward-compatibility.